### ISSUE TITLE
Hide Movie Genres

### ISSUE THREAD

**Issue by AkikoKumagara** [Radiicall/jellyfin-rpc#120]
### Is there an existing issue for this?

- [X] I have searched the existing feature requests

### Is your feature request related to a problem? Please describe

Yes, movie genres are treated the same way as music artists, which doesn't make a ton of sense to me. As such there's no way to disable them from displaying them that I can see. I'm hoping this isn't something at the fault of Jellyfin itself, but I suppose that's possible.

![image](https://github.com/Radiicall/jellyfin-rpc/assets/15231336/3e547b93-53db-4759-9638-9beeace9c547)

### Describe the solution you'd like

Ability to hide movie genres from displaying; perhaps making them work the same way as music genres, _if possible_.
Thanks for your hard work in maintaining and improving this 🙇‍♀️

**Comment by Radiicall** [comment]
I know this issue has been open for a while and it will probably stay open for a while longer, im working on a total refactor which should in theory make implementing new features easier

**Comment by AkikoKumagara** [comment]
> I know this issue has been open for a while and it will probably stay open for a while longer, im working on a total refactor which should in theory make implementing new features easier

That's ok. Thanks for the update. Grateful for what you do here.

**Comment by Radiicall** [comment]
![image](https://github.com/user-attachments/assets/340ddccf-7eba-4a02-b07e-fd2d426d38c1)
are you thinking something customizable like this?
It works in the same way as the music config.

```json
    "music": {
      "display": [
        "year",
        "album",
        "genres"
      ],
      "separator": null
    },
    "movies": {
      "display": [
        "year",
        "critic-score",
        "community-score",
        "genres"
      ],
      "separator": null
    },
```

**Comment by AkikoKumagara** [comment]
Oh, that's awesome.

**Comment by Radiicall** [comment]
If you want to test this functionality then you can download the latest artifact from the bottom of [this page](https://github.com/Radiicall/jellyfin-rpc/actions/runs/10021576934?pr=125)

**Comment by Radiicall** [comment]
Fixed in #125

**Comment by AkikoKumagara** [comment]
> Fixed in #125

I'm trying on Windows with the new release and this is the only output with no Rich Presence display?
![image](https://github.com/user-attachments/assets/281d8dd9-3ff1-4b75-a4d1-98726aeb24ea)

Config looks like this:
`{
  "jellyfin": {
    "url": "http://xxxxxxxxxxxxxxxxx",
    "api_key": "xxxxxxxxxxxxxxxxxxxxx",
    "username": [
      "XXXXX"
    ],
    "movies": {
      "display": [],
      "separator": "-"
    },
    "music": {
      "display": [],
      "separator": "—"
    },
    "blacklist": null,
    "self_signed_cert": null
  },
  "discord": {
	"show_paused": false,
    "application_id": "XXXXXXXXXXXXXX",
    "buttons": [
            {
                "name": "The Bear's Den",
                "url": "https://thebearsden.live"
            },
            {
                "name": "dynamic",
                "url": "dynamic"
            }
        ]
  },
  "imgur": {
    "client_id": "XXXXXXXXXXXXX"
  },
  "images": {
    "enable_images": true,
    "imgur_images": true
  }
}`
Put x's in place of sensitive data.
I'm not sure what's going on. The same config works on the previous version fine. Am I missing something here?
